### PR TITLE
DCOS-8068 Add health check results table to task page

### PR DESCRIPTION
**Pull request by pierlo-upitup**
This adds the missing Health Check Results table to the task detail page as is available in Marathon UI.

image

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @aldipower

**Comment by orlandohohmeier** [comment]
LGTM 💯 ✨

**Comment by pierlo-upitup** [comment]
retest this please

File: src/js/components/MarathonTaskDetailsList.js [hunk 1]
```diff
@@ -89,10 +95,54 @@ class MarathonTaskDetailsList extends React.Component {
...
   }
 
+  getMarathonTaskHealthCheckResults(task) {
+    if (task == null || task.healthCheckResults == null) {
+
+      return null;
+    }
+
+    return task.healthCheckResults.map((result, i) =>{
+      let consecutiveFailures = result.consecutiveFailures;
```
Comment (kennyt) [inline src/js/components/MarathonTaskDetailsList.js]: is it possible to do a `|| 'None'` here?

**Comment by kennyt** [comment]
retest this please

**Comment by kennyt** [comment]
retest this please

**Comment by kennyt** [comment]
merge once super nit is fixed

File: src/js/components/MarathonTaskDetailsList.js (hunk 1 above)
Comment (pierlo-upitup) [inline src/js/components/MarathonTaskDetailsList.js]: nope, this can be a `0`.

**Comment by pierlo-upitup** [comment]
relevant tests are passing 
image

**Pull request merged**